pulp_smash/pulp3/utils.py

```diff
@@ -139,6 +139,27 @@ def publish(cfg, publisher, repo, version_href=None):
     return client.get(tasks[-1]["created_resources"][0])
 
 
+def publication(cfg, path, repo, version_href=None, **kwargs):
```

Comment:
Besides @rochacbruno comments.

path is not listed in the docstrings.

**kwargs not used in the code.